src/Action/Delivery/CalculateRetailPrice.php

```diff
@@ -52,7 +54,8 @@ public function __invoke(Delivery $data, Request $request)
         $amount = $this->deliveryManager->getPrice($data, $store->getPricingRuleSet());
 
         if (null === $amount) {
-            throw new BadRequestHttpException('Price could not be calculated');
+            $message = $this->translator->trans('delivery.price.error.priceCalculation', [], 'validators');
+            throw new BadRequestHttpException($message);
```

Comment:
I think it should return a code like `PRICE_CALCULATION_ERROR` with the actual translation provided on a client side